### ISSUE TITLE
how to use pre-trained model to continue trainning？

### ISSUE THREAD

**Issue by obitooooo** [lucastabelini/LaneATT#119]
thanks for your great code！
i have a problem when i learning lane detection，about laneATT，i downloaded the ‘laneatt_experiments’，it contain some experiments pre-trained model ，it use them to detect some lane video，now i wanna how to use them to continue training，because retrain cost a lot of time，and I have a smaller datasets，so i just want to do some fine tune。
Looking forward to your reply，thanks！

**Comment by lucastabelini** [comment]
You can modify the source code to load the pretrained model and then just train as usual (maybe with a reduced learning rate).
